draft-ietf-httpbis-rand-access-live.xml

```diff
@@ -238,16 +239,18 @@
       representation length) may respond with a 206 status code (Partial Content) with a Content-Range
       header indicating the currently satisfiable byte range. For example:
       </t>
-<figure><artwork type="example">
-      HEAD /resource HTTP/1.1
-      Range: bytes=0-
+<figure><artwork type="message/http; msgtype=&#34;request&#34;" x:indent-with="    ">
+HEAD /resource HTTP/1.1
+Host: example.com
+Range: bytes=0-
+
```

Comment:
Do you think the blank lines at the end of the examples are really necessary? (Presume the intent is to show the terminating \n\n?)

Reply:
We do it in the base specs as well...

Reply:
Good enough reason for me. Consistency is good....